### PR TITLE
fixed merge with footer

### PR DESCRIPTION
**Pull request by margeschrec**
fixed duplicate footer divs that was a result of a merge mistake from last pull request.

### CODE REVIEW

**Review by billythakidd04 — DISMISSED** [review]

**Review by vricci518 — CHANGES_REQUESTED** [review]
Deleting rows 26 thru 43 will remove the duplicate locations. But the remaining locations will not have the responsive code so when viewed on a mobile device the locations appear one across rather than 3.

Each locations should be enclosed with the following:
code from line 26 before each location code

and then closing div after each location code

This will make the locations populate 3 across on a large screen and 1 across on a small screen.

**Comment by billythakidd04** [comment]
> Deleting rows 26 thru 43 will remove the duplicate locations. But the remaining locations will not have the responsive code so when viewed on a mobile device the locations appear one across rather than 3.
> 
> Each locations should be enclosed with the following:
> code from line 26 before each location code
> 
> and then closing div after each location code
> 
> This will make the locations populate 3 across on a large screen and 1 across on a small screen.

YUP! I didn't realize this was the main footer file! Nice catch @vricci518 !

**Comment by margeschrec** [comment]
> Deleting rows 26 thru 43 will remove the duplicate locations. But the remaining locations will not have the responsive code so when viewed on a mobile device the locations appear one across rather than 3.
> 
> Each locations should be enclosed with the following:
> 
> before the [ and then ](....)
> [ after each ](....)
> This will make the locations populate 3 across on a large screen and 1 across on a small screen.


gotcha! All fixed., nice catch.

**Review by vricci518 — APPROVED** [review]
looks good now

File: footer.php [hunk 1]
```diff
@@ -42,4 +29,5 @@
 			</a>
 		</div>
 	</div>
+	</div>
```
Comment (billythakidd04) [inline footer.php]: extra closing div tag

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
extra div tag

**Review by billythakidd04 — APPROVED** [review]

**Review by vricci518 — APPROVED** [review]

**Pull request merged**